pt-BR/1-5.md

```diff
@@ -0,0 +1,86 @@
+
+Esta propriedade será uma string - o nome da imagem a ser carregada - mas ela precisa ser uma string *opcional* porque quando o view controller for criado inicialmente ele não saberá o valor da propriedade. Logo após a criação ela será preenchida, mas ainda assim, quando o view controller for inicializado ela estará vazia.
+
+Então, adicione esta propriedade ao `DetailViewController`, logo após o `@IBOutlet`:
+
+    var selectedImage: String?
+
+A primeira tarefa foi realizada, vamos para a próxima: implementar `didSelectRowAt` de maneira a carregar o `DetailViewController` a partir do storyboard.
+
+Quando nós criamos o view controller de detalhes, você preencheu o storyboardID "Detail", o que nos permite carregar ele a partir do storyboard usando um método com o nome `instantiateViewController(withIdentifier:)`. Todos view controllers possuem uma propriedade chamada `storyboard` que faz referência ao storyboard pelo qual o view controller foi instanciado, ou nil. No caso do `ViewController` esta referência será ao Main.storyboard, que é o mesmo storyboard que contém o detail view controller. Por isto, vamos usar esta referência para chamá-lo.
+
+Nós podemos dividir esta tarefa em três tarefas menores, sendo duas delas novas:
+
+1. Carregar o layout do `DetailViewController` a partir do storyboard.
+2. Definir a propriedade `selectedImage` do `DetailViewController` com o item selecionado, a partir do array `pictures`.
+3. Mostrar o novo view controller.
+
+A primeira das mini-tarefas é realizada chamando `instantiateViewController`, mas existem duas pequenas complicações. Primeiro nós executamos este método usando a propriedade `storyboard` que esta presente nos objetos do tipo `UIViewController`. Entretanto este método é um opcional porque existe a possibilidade que o objeto deste tipo não tenha sido inicializado a partir de um storyboard. Por isto, temos que usar `?`, como no caso em que alteramos o valor do texto da nossa célula: "tente alterar este valor, mas não faça nada caso encontre algum problema."
+
+Segundo, mesmo tendo certeza que o método `instantiateViewController()` irá nos responder com `DetailViewController`, Swift *acredita* que ele irá nos responder com um `UIViewController` porque a linguagem não consegue interpretar qual o tipo exato que foi descrito dentro do storyboard.
+
+Isto parecerá confuso se você é novo em programação, por isso, deixe-me tentar te explicar melhor usando uma analogia. Vamos dizer que você tem um encontro hoje a noite, diante disto você me pede para lhe conseguir dois ingressos para um evento. Sendo seu amigo eu saio e procuro os ingressos, coloco-os em um envelope e lhe entrego. Eu realizei a minha parte do combinado: Você pediu os ingressos, recebeu os ingressos. Mas que tipo de ingressos são? Ingressos para o teatro? Para um jogo de esportes? Para o metrô? A única maneira de você saber é abrir o envelope e conferir.
```

Comment:
typo: hoje à noite instead hoje a noite. 
I'm sorry, I'm Portuguese and sometimes I 'm not sure about pt-BR grammar.
Please ignore comments if not make sense to you.

Reply:
In that case I'm pretty sure that it does not apply

Reply:
Ignore comment